infra/resources/_modules/functions_apps/app_service_plan_wallet_common.tf

```diff
@@ -0,0 +1,16 @@
+resource "azurerm_app_service_plan" "app_service_plan_wallet_common" {
```

Comment:
resource deprecated, use https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/service_plan

Reply:
add also zone_balancing_enabled = true and min instances to 3

Reply:
Done in https://github.com/pagopa/io-wallet/pull/56/commits/9aca3808ab1fb50c4b5b792a2d6efa5b174ecd2b.